src/main/frontend/handler/editor.cljs

```diff
@@ -399,7 +398,7 @@
         (merge (if level {:block/level level} {})))))
 
 (defn- save-block-inner!
-  [repo block value {}]
+  [_repo block value {}]
```

Comment:
Would encourage us to remove unused arguments. In this case, looks like `save-block-if-changed!` calls `state/get-current-repo` for no reason